### PR TITLE
ci: write report comment instead of writing to PR description

### PR DESCRIPTION
**Pull request by akhdrv**
## Release Notes

Отчет перфтестов писать в комментах, а не в шапке PR, как было ранее.
Тест в https://github.com/salute-developers/plasma/pull/839

### What/why Changed

Писать отчет в шапку PR оказалось плохой идеей, поскольку технически сложно гарантировать отсутствие коллизий с release джобой. Синхронизация затруднительна из-за того, что мы поддерживаем не весь код, который пишет в шапку.

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.838.6824194830.ts'>borderRadius_default_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.838.6824194830.ts'>borderRadius_plasma_b2c_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.838.6824194830.ts'>borderRadius_plasma_web_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.838.6824194830.ts'>borderRadius_sberHealth_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.838.6824194830.ts'>borderRadius_sbermarket_business_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.838.6824194830.ts'>borderRadius_sbermarket_metro_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.838.6824194830.ts'>borderRadius_sbermarket_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.838.6824194830.ts'>borderRadius_sbermarket_selgros_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.838.6824194830.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.838.6824194830.ts'>borderRadius_sberonline_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.838.6824194830.ts'>borderRadius_sberprime_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.838.6824194830.ts'>borderRadius_stylesSalute_react-native--canary.838.6824194830.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.838.6824194830.swift'>color_default_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.838.6824194830.kt'>color_default_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.838.6824194830.ts'>color_default_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.838.6824194830.xml'>color_default_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.838.6824194830.swift'>color_plasma_b2c_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.838.6824194830.kt'>color_plasma_b2c_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.838.6824194830.ts'>color_plasma_b2c_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.838.6824194830.xml'>color_plasma_b2c_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.838.6824194830.swift'>color_plasma_web_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.838.6824194830.kt'>color_plasma_web_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.838.6824194830.ts'>color_plasma_web_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.838.6824194830.xml'>color_plasma_web_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.838.6824194830.swift'>color_sberHealth_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.838.6824194830.kt'>color_sberHealth_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.838.6824194830.ts'>color_sberHealth_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.838.6824194830.xml'>color_sberHealth_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.838.6824194830.swift'>color_sbermarket_business_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.838.6824194830.kt'>color_sbermarket_business_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.838.6824194830.ts'>color_sbermarket_business_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.838.6824194830.xml'>color_sbermarket_business_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.838.6824194830.swift'>color_sbermarket_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.838.6824194830.kt'>color_sbermarket_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.838.6824194830.swift'>color_sbermarket_metro_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.838.6824194830.kt'>color_sbermarket_metro_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.838.6824194830.ts'>color_sbermarket_metro_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.838.6824194830.xml'>color_sbermarket_metro_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.838.6824194830.ts'>color_sbermarket_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.838.6824194830.swift'>color_sbermarket_selgros_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.838.6824194830.kt'>color_sbermarket_selgros_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.838.6824194830.ts'>color_sbermarket_selgros_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.838.6824194830.xml'>color_sbermarket_selgros_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.838.6824194830.swift'>color_sbermarket_wlbusiness_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.838.6824194830.kt'>color_sbermarket_wlbusiness_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.838.6824194830.ts'>color_sbermarket_wlbusiness_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.838.6824194830.xml'>color_sbermarket_wlbusiness_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.838.6824194830.xml'>color_sbermarket_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.838.6824194830.swift'>color_sberonline_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.838.6824194830.kt'>color_sberonline_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.838.6824194830.ts'>color_sberonline_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.838.6824194830.xml'>color_sberonline_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.838.6824194830.swift'>color_sberprime_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.838.6824194830.kt'>color_sberprime_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.838.6824194830.ts'>color_sberprime_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.838.6824194830.xml'>color_sberprime_xml--canary.838.6824194830.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.838.6824194830.swift'>color_stylesSalute_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.838.6824194830.kt'>color_stylesSalute_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.838.6824194830.ts'>color_stylesSalute_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.838.6824194830.xml'>color_stylesSalute_xml--canary.838.6824194830.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.838.6824194830.ts'>shadow_default_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.838.6824194830.ts'>shadow_plasma_b2c_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.838.6824194830.ts'>shadow_plasma_web_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.838.6824194830.ts'>shadow_sberHealth_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.838.6824194830.ts'>shadow_sbermarket_business_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.838.6824194830.ts'>shadow_sbermarket_metro_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.838.6824194830.ts'>shadow_sbermarket_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.838.6824194830.ts'>shadow_sbermarket_selgros_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.838.6824194830.ts'>shadow_sbermarket_wlbusiness_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.838.6824194830.ts'>shadow_sberonline_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.838.6824194830.ts'>shadow_sberprime_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.838.6824194830.ts'>shadow_stylesSalute_react-native--canary.838.6824194830.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.838.6824194830.ts'>spacing_default_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.838.6824194830.ts'>spacing_plasma_b2c_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.838.6824194830.ts'>spacing_plasma_web_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.838.6824194830.ts'>spacing_sberHealth_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.838.6824194830.ts'>spacing_sbermarket_business_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.838.6824194830.ts'>spacing_sbermarket_metro_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.838.6824194830.ts'>spacing_sbermarket_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.838.6824194830.ts'>spacing_sbermarket_selgros_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.838.6824194830.ts'>spacing_sbermarket_wlbusiness_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.838.6824194830.ts'>spacing_sberonline_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.838.6824194830.ts'>spacing_sberprime_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.838.6824194830.ts'>spacing_stylesSalute_react-native--canary.838.6824194830.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.838.6824194830.swift'>typo_mage_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.838.6824194830.kt'>typo_mage_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.838.6824194830.ts'>typo_mage_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.838.6824194830.swift'>typo_plasma_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.838.6824194830.kt'>typo_plasma_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.838.6824194830.ts'>typo_plasma_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.838.6824194830.swift'>typo_ruler_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.838.6824194830.kt'>typo_ruler_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.838.6824194830.ts'>typo_ruler_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.838.6824194830.swift'>typo_sage_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.838.6824194830.kt'>typo_sage_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.838.6824194830.ts'>typo_sage_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.838.6824194830.swift'>typo_sbermarket_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.838.6824194830.kt'>typo_sbermarket_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.838.6824194830.ts'>typo_sbermarket_react-native--canary.838.6824194830.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.838.6824194830.swift'>typo_soulmate_ios-swift--canary.838.6824194830.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.838.6824194830.kt'>typo_soulmate_kotlin--canary.838.6824194830.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.838.6824194830.ts'>typo_soulmate_react-native--canary.838.6824194830.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-838/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-838/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-838/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-838/

File: actions/perftest-helpers/src/modules/writeComment.test.ts [hunk 1]
```diff
@@ -44,155 +42,58 @@ describe('perftest/updateDescription', () => {
...
-                pull_number: prId,
-            };
-            const updatedDescription = 'updated description';
 
-            getMock.mockResolvedValue(prData as never);
             readJsonMock.mockResolvedValue(report);
-            getUpdatedDescriptionMock.mockReturnValue(updatedDescription);
-            updateMock.mockResolvedValue(null as never);
+            getCommentBodyMock.mockReturnValue(commentBody);
+            // createCommentMock.mockResolvedValue(null as never);
```
Comment (Yakutoc) [inline actions/perftest-helpers/src/modules/writeComment.test.ts:50]: @akhdrv А этот закомментированный код нужен? Если нет, убери его пожалуйста, чтобы потом голову не ломать почему его  закомментировали )

**Review by TitanKuzmich — APPROVED** [review]

File: actions/perftest-helpers/src/modules/writeComment.test.ts (hunk 1 above)
Comment (akhdrv) [reply actions/perftest-helpers/src/modules/writeComment.test.ts]: @Yakutoc дай лайк

**Review by kayman233 — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-838/

**Review by Yakutoc — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-838/

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version:  🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version:  🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version:  🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.3.0-dev.0, @salutejs/plasma-b2c@1.244.0-dev.0, @salutejs/plasma-core@1.139.0-dev.0, @salutejs/plasma-hope@1.242.0-dev.0, @salutejs/plasma-icons@1.171.0-dev.0, @salutejs/plasma-new-hope@0.12.0-dev.0, @salutejs/plasma-sb-utils@0.137.0-dev.0, @salutejs/plasma-temple-docs@0.181.0-dev.0, @salutejs/plasma-temple@1.189.0-dev.0, @salutejs/plasma-ui-docs@0.230.0-dev.0, @salutejs/plasma-ui@1.221.0-dev.0, @salutejs/plasma-web-docs@0.198.0-dev.0, @salutejs/plasma-web@1.244.0-dev.0, @salutejs/plasma-website@0.206.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.4.0-dev.0, @salutejs/plasma-b2c@1.245.0-dev.0, @salutejs/plasma-hope@1.243.0-dev.0, @salutejs/plasma-temple-docs@0.182.0-dev.0, @salutejs/plasma-ui-docs@0.231.0-dev.0, @salutejs/plasma-web-docs@0.199.0-dev.0, @salutejs/plasma-web@1.245.0-dev.0, @salutejs/plasma-website@0.207.0-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-asdk@0.5.0-dev.0, @salutejs/plasma-b2c@1.246.0-dev.0, @salutejs/plasma-hope@1.244.0-dev.0, @salutejs/plasma-new-hope@0.13.0-dev.0, @salutejs/plasma-temple-docs@0.183.0-dev.0, @salutejs/plasma-ui-docs@0.232.0-dev.0, @salutejs/plasma-web-docs@0.200.0-dev.0, @salutejs/plasma-web@1.246.0-dev.0, @salutejs/plasma-website@0.208.0-dev.0 🚀

**Pull request merged**